### ISSUE TITLE
Can you share pretrained?

### ISSUE THREAD

**Issue by chunping-xt** [julianstefinovic/DanceGPT2#1]
The result is very interesting, can you share the pretrained?
Do you have any plans to expand this project further, for example creating music with prompts: text, midi,...?

**Comment by julianstefinovic** [comment]
Hi, thanks for your interest in the project! Currently, I don't have any plans to expand the project but I might look into it at some point in the future. With regard to the pretrained weights, I tried to upload them directly via GitHub but I would need to setup git-lfs in order to upload them since the file is ~300MB, and it seems a little involved on Windows. I've tried to also upload them via HuggingFace, since they provide git-lfs by default but the `model.safetensors` file gets stuck on "Hashing" when uploading it 😅. I'll let you know if I have any luck uploading the weights 🙂

**Comment by julianstefinovic** [comment]
Hi, thanks for your interest in the project! Currently, I don't have any plans to expand the project but I might look into it at some point in the future. With regard to the pretrained weights, I tried to upload them directly via GitHub but I would need to setup git-lfs in order to upload them since the file is ~300MB, and it seems a little involved on Windows. I've tried to also upload them via HuggingFace, since they provide git-lfs by default but the `model.safetensors` file gets stuck on "Hashing" when uploading it 😅. I'll let you know if I have any luck uploading the weights 🙂

**Comment by julianstefinovic** [comment]
Hi again, I was able to upload the model weights onto [a HuggingFace repository](https://huggingface.co/JulianS/DanceGPT2/tree/main). I have also updated the README with instructions for loading in the model for inference 😄

**Comment by chunping-xt** [comment]
Thank you for your effort in sharing pretrained, it seems you still forgot to upload the config file.

### load model
from transformers import GPT2LMHeadModel, GPT2Tokenizer
model_path = 'JulianS/DanceGPT2'
model = GPT2LMHeadModel.from_pretrained(model_path)
\>
OSError: JulianS/DanceGPT2 does not appear to have a file named config.json. Checkout 
...

### encodec module
Appreciate if you share your codes of encodec.py or class encodec, I have to use code from transformers to load encodec:
from transformers import EncodecModel, AutoProcessor
enc_model = EncodecModel.from_pretrained("facebook/encodec_48khz")
processor = AutoProcessor.from_pretrained("facebook/encodec_48khz")

but I'm not sure it will work with your part of the code :)

**Comment by julianstefinovic** [comment]
Hi again,

> Thank you for your effort in sharing pretrained, it seems you still forgot to upload the config file.

I have loaded the config files into the HuggingFace repository, so hopefully the model can be loaded in correctly now.

> Appreciate if you share your codes of encodec.py or class encodec, I have to use code from transformers to load encoded

On my implementation, I installed the encodec library via `!pip install encodec`. Otherwise, the library should also be obtainable via the official [repository](https://github.com/facebookresearch/encodec).

Hope this helps!
